content/doc/book/installing/_jenkins-command-parameters.adoc

```diff
@@ -50,7 +50,7 @@ to 60 minutes
 
 === Miscellaneous parameters
 
-Other Jenkins initialization configuration is also controlled by command line arguments.
+Other Jenkins initialization configurations are also controlled by command line arguments.
```

Comment:
I think a "configuration" usually refers to a collection of options, stored in e.g. a "configuration file". So the use of "configurations" here seems off to me, because there is only one collection of command-line options. Perhaps it might be better to phrase this as

```suggestion
Other Jenkins initialization options are also controlled by command line arguments.
```

That is, multiple options [in a single configuration] and controlled by command line arguments.

Reply:
Thanks for clarifying that for me @basil, options definitely seems like the right usage in this case.